api/controller/instancepoller/machine_test.go

```diff
@@ -52,7 +53,7 @@ func (s *MachineSuite) TestNonFacadeMethods(c *gc.C) {
 
 // methodWrapper wraps a Machine method call and returns the error,
 // ignoring the result (if any).
-type methodWrapper func(*instancepoller.Machine) error
+type methodWrapper func(*instancepoller.Machine, context.Context) error
```

Comment:
Flip the order of the arguments. Keep govet happy.

Reply:
Sadly machine is the receiver and we are using a method on it as a static method so the arg order is fixed.